vnext/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.windows.js

```diff
@@ -39,147 +65,286 @@ const _subscriptions = new Map();
  *
  * See https://reactnative.dev/docs/accessibilityinfo.html
  */
-
 const AccessibilityInfo = {
   /**
-   * iOS only
+   * Query whether bold text is currently enabled.
+   *
+   * Returns a promise which resolves to a boolean.
+   * The result is `true` when bold text is enabled and `false` otherwise.
+   *
+   * See https://reactnative.dev/docs/accessibilityinfo.html#isBoldTextEnabled
    */
-  isBoldTextEnabled: function(): Promise<boolean> {
-    return Promise.resolve(false);
+  isBoldTextEnabled(): Promise<boolean> {
+    if (Platform.OS === 'android') {
```

Comment:
Make sure that all the if-else blocks in this file will call something correct on Windows.